.github/workflows/backend-api-spec-update.yml

```diff
@@ -0,0 +1,42 @@
+name: Update backend api spec generated by Kotlin Gradle Plugin
+
+on:
+  pull_request:
+    paths:
+      - 'save-backend/**/*Controller'
```

Comment:
Should it be `*Controller.kt` or `*Controller*` or does it match this as a substring?

Reply:
Fixed